Userland/Libraries/LibGUI/AbstractTableView.cpp

```diff
@@ -422,7 +422,7 @@ void AbstractTableView::layout_headers()
         int y = frame_thickness() + column_header_height - vertical_scrollbar().value();
         int height = max(content_height(), rect().height() - frame_thickness() * 2 - column_header_height - horizontal_scrollbar_height);
 
-        row_header().set_relative_rect(x, y, row_header().min_size().width(), height);
+        row_header().set_relative_rect(x, y, row_header().effective_min_size().width().value_verify_regular(), height);
```

Comment:
min_size -> effective_min_size and adding value_verify_regular is one commit

Reply:
Are you trying to say I should split those two changes into separate commits?

Reply:
Yes, maybe UIDim can VERIFY when it returns non-standard sizes for a few commits, then you could split these 2 changes

Reply:
I went with a different way, hopefully it works just as well.